src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,469 @@
+                        'title' => $this->__('Caching')
+                    )
+                ),
+                'theme' => array(
+                    'compile_check' => array(
+                        'state' => $modvars['compile_check'],
+                        'title' => $this->__('Compile check')
+                    ),
+                    'force_compile' => array(
+                        'state' => $modvars['force_compile'],
+                        'title' => $this->__('Force compile')
+                    ),
+                    'cache' => array(
+                        'state' => $modvars['enablecache'],
+                        'title' => $this->__('Caching')
+                    )
+                )
+            );
+        }
+        return $this->render("ZikulaAdminModule:AdminInterface:developernotices.html.twig", [
+            'developer' => $data
+        ]);
+    }
+
+    /**
+     * @Route("/securityanalyzer")
+     *
+     * Add security analyzer
+     *
+     * @return Response symfony response object
+     */
+    public function securityanalyzerAction()
+    {
+        $data = [];
+        $data['legacy'] = array(
+            // check for magic_quotes - this was removed php 5.4.0
+            'magic_quotes_gpc' => get_magic_quotes_gpc(),
+            // check for register_globals - this was removed php 5.4.0
+            'register_globals' => ini_get('register_globals'),
+            // check for config.php being writable - this is deprecated
+            'config_php' => (bool) is_writable('config/config.php')
+        );
+        // check for .htaccess in temp directory
+        $app_htaccess = false;
+        $appDir = $this->get('kernel')->getRootDir();
+        if ($appDir) {
+            // check if we have an absolute path which is possibly not within the document root
+            $docRoot = $this->get('request_stack')->getMasterRequest()->server->get('DOCUMENT_ROOT');
+            if (\StringUtil::left($appDir, 1) == '/' && (strpos($appDir, $docRoot) === false)) {
+                // temp dir is outside the webroot, no .htaccess file needed
+                $app_htaccess = true;
+            } else {
+                if (strpos($appDir, $docRoot) === false) {
+                    $ldir = dirname(__FILE__);
+                    $p = strpos($ldir, DIRECTORY_SEPARATOR . 'system'); // we are in system/Admin
+                    $b = substr($ldir, 0, $p);
+                    $filePath = $b . '/' . $appDir . '/.htaccess';
+                } else {
+                    $filePath = $appDir . '/.htaccess';
+                }
+                $app_htaccess = (bool) file_exists($filePath);
+            }
+        } else {
+            // already customized, admin should know about what he's doing...
+            $app_htaccess = true;
+        }
+        $data['app_htaccess'] = $app_htaccess;
+        
+        $data['scactive'] = (bool) \ModUtil::available('ZikulaSecurityCenterModule');
```

Comment:
I have re-considered `ModUtil::available()` and now think that we should use this until Core-2.0 at which time ALL Core-2.0 spec modules (and themes) will be bundles and loaded into the kernel so checking the kernel will be all that is needed. ping @Guite @cmfcmf